fossunited/api/schedule.py

```
@@ -97,7 +104,225 @@ def get_event_schedule(event_id: str) -> dict:
         date: dict(halls)
         for date, halls in sorted(
-            schedule_by_date_and_hall.items(), key=lambda x: datetime.strptime(x[0], "%d/%m/%Y")
+            schedule_by_date_and_hall.items(),
+            key=lambda x: datetime.strptime(x[0], "%d/%m/%Y"),
         )
     }
     return sorted_schedule
+
+
+@frappe.whitelist(allow_guest=True)
+def download_schedule(event, format="ics", days="", halls=""):
```

Ruff (0.12.2)

Unused function argument: `days`

(ARG001)